### PR TITLE
[Issue #18055] The document is not ture about sharding rules

### LINKED ISSUE #18055: The 5.1.1 document is not ture about sharding rules
### Which version of ShardingSphere did you use?
5.1.1
### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
ShardingSphere-JDBC
### Expected behavior
configure sharding-jdbc properties
### Reason analyze (If you can)
the document is not update 
### Example codes for reproduce this issue (such as a github link).

image

I use this document configuration,but it is not work.
So I check the source code from example and find it likes 
image

Is the document need to be update?
the source code link [application-sharding-databases.properties](https://github.com/apache/shardingsphere/blob/master/examples/shardingsphere-jdbc-example/single-feature-example/sharding-example/sharding-spring-boot-jpa-example/src/main/resources/application-sharding-databases.properties)

### PR DESCRIPTION
**Pull request by 726411951**
Fixes #18055.

Changes proposed in this pull request:
- add '.rules' in /docs

### CODE REVIEW

File: docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.cn.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ spring.shardingsphere.rules.sharding.broadcast-tables[0]= # 广播表规则列
 spring.shardingsphere.rules.sharding.broadcast-tables[1]= # 广播表规则列表
 spring.shardingsphere.rules.sharding.broadcast-tables[x]= # 广播表规则列表
 
-spring.shardingsphere.sharding.default-database-strategy.xxx= # 默认数据库分片策略
-spring.shardingsphere.sharding.default-table-strategy.xxx= # 默认表分片策略
-spring.shardingsphere.sharding.default-key-generate-strategy.xxx= # 默认分布式序列策略
+spring.shardingsphere.rules.sharding.default-database-strategy.xxx= # 默认数据库分片策略
+spring.shardingsphere.rules.sharding.default-table-strategy.xxx= # 默认表分片策略
+spring.shardingsphere.rules.sharding.default-key-generate-strategy.xxx= # 默认分布式序列策略
```
Comment (terrymanu) [inline docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.cn.md:54]: This is the change history, is it need to modify?

File: docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.en.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ spring.shardingsphere.rules.sharding.broadcast-tables[0]= # Broadcast tables
 spring.shardingsphere.rules.sharding.broadcast-tables[1]= # Broadcast tables
 spring.shardingsphere.rules.sharding.broadcast-tables[x]= # Broadcast tables
 
-spring.shardingsphere.sharding.default-database-strategy.xxx= # Default strategy for database sharding
-spring.shardingsphere.sharding.default-table-strategy.xxx= # Default strategy for table sharding
-spring.shardingsphere.sharding.default-key-generate-strategy.xxx= # Default Key generator strategy
+spring.shardingsphere.rules.sharding.default-database-strategy.xxx= # Default strategy for database sharding
+spring.shardingsphere.rules.sharding.default-table-strategy.xxx= # Default strategy for table sharding
+spring.shardingsphere.rules.sharding.default-key-generate-strategy.xxx= # Default Key generator strategy
```
Comment (terrymanu) [inline docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.en.md:54]: This is the change history, is it need to modify?

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.cn.md (hunk 1 above)
Comment (726411951) [reply docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.cn.md:54]: Yes, it's used in '/examples' like [Line 34](https://github.com/apache/shardingsphere/blob/c89217636d579a050d59a95e0e532980d5488dd3/examples/shardingsphere-jdbc-example/single-feature-example/extension-example/custom-sharding-algortihm-example/spi-based-sharding-algorithm-example/spi-based-sharding-spring-boot-jpa-example/src/main/resources/application-sharding-databases-tables.properties)

File: docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.en.md (hunk 2 above)
Comment (726411951) [reply docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.en.md:54]: Yes, it's used in '/examples' like [Line 34](https://github.com/apache/shardingsphere/blob/c89217636d579a050d59a95e0e532980d5488dd3/examples/shardingsphere-jdbc-example/single-feature-example/extension-example/custom-sharding-algortihm-example/spi-based-sharding-algorithm-example/spi-based-sharding-spring-boot-jpa-example/src/main/resources/application-sharding-databases-tables.properties)

Comment (terrymanu) [reply docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.en.md:54]: What I mean is the doc is for history version, not for current version

Comment (726411951) [reply docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.en.md:54]: I got it, it is not necessary to modify

**Comment by 726411951** [comment]
@terrymanu  Hi，I  have reverted the changes about  history files

**Review by terrymanu — APPROVED** [review]

**Pull request merged**